lib/src/solver/report.dart

```diff
@@ -130,6 +130,28 @@ class SolveReport {
     }
   }
 
+  /// Displays a two-line message, number of outdated packages and an
+  /// instruction to run `pub outdated` if outdated packages are detected.
+  void reportOutdated() {
+    final outdatedPackagesCount = _result.packages.where((id) {
+      final versions = _result.availableVersions[id.name];
+      return versions.any((v) =>
+          v > id.version &&
+          (id.version.isPreRelease == true || v.isPreRelease == false));
```

Comment:
Would be nice with a comment documenting what this logic is supposed to mean.

Reply:
Done!